Framework/Core/src/CommonDataProcessors.cxx

```diff
@@ -399,32 +399,57 @@ DataProcessorSpec
     auto& callbacks = ic.services().get<CallbackService>();
     callbacks.set(CallbackService::Id::EndOfStream, endofdatacb);
 
+    // prepare map<uint64_t, uint64_t>(startTime, TFNumber)
+    std::map<uint64_t, uint64_t> TFNumbers;
```

Comment:
please make sure variables always start with lowercase letters.  I would use `tfNumbers` or `timeframeNumbers` in this particular case.

Reply:
done